packages/reactive-element/src/reactive-element.ts

```diff
@@ -37,6 +37,7 @@ const {
   getOwnPropertyDescriptor,
   getOwnPropertyNames,
   getOwnPropertySymbols,
+  getPrototypeOf,
```

Comment:
Is this smaller? It's writing getPrototypeOf twice instead of once to save an `Object`

Reply:
It's only writing it once - the second reference can be renamed.

Because we're already tearing off Object statics, I think even one use pays for itself. This block is rewritten to something like:

```ts
const{is:e,defineProperty:t,getOwnPropertyDescriptor:r,getOwnPropertyNames:o,getOwnPropertySymbols:p,getPrototypeOf:y}=Object;
```

So the total cost is `,getPrototypeOf:y` + `y(this)` instead of `Object.getPrototypeOf(this)`, so it's essentially a minification of just `Object.` to whatever the variable is.

The other benefit is that there isn't an object dereference of `.getPrototypeOf` at the call site.